data_steward/resources/fields/post_deid_person.json

```diff
@@ -0,0 +1,128 @@
+[
```

Comment:
This file can cause issues in `resources.cdm_schemas` if it is not excluded within that function.

Reply:
updated